### PR TITLE
fix(site): the img element of site should not be block

### PR DESCRIPTION
**Pull request by li-jia-nan**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE_CN.md?plain=1)

### 🤔 This is a ...

- [ ] 🆕 New feature
- [x] 🐞 Bug fix
- [x] 📝 Site / documentation improvement
- [ ] 📽️ Demo improvement
- [ ] 💄 Component style improvement
- [ ] 🤖 TypeScript definition improvement
- [ ] 📦 Bundle size optimization
- [ ] ⚡️ Performance optimization
- [ ] ⭐️ Feature enhancement
- [ ] 🌐 Internationalization
- [ ] 🛠 Refactoring
- [ ] 🎨 Code style optimization
- [ ] ✅ Test Case
- [ ] 🔀 Branch merge
- [ ] ⏩ Workflow
- [ ] ❓ Other (about what?)

### 🔗 Related Issues

> - Describe the source of related requirements, such as links to relevant issue discussions.
> - For example: close #xxxx, fix #xxxx

### 💡 Background and Solution

> - The specific problem to be addressed.
> - List the final API implementation and usage if needed.
> - If there are UI/interaction changes, consider providing screenshots or GIFs.

### 📝 Change Log

> - Describe the impact of the changes on developers, not the solution approach.
> - Reference: https://ant.design/changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | fix(site): the img element of site should not be block element |
| 🇨🇳 Chinese | fix(site): the img element of site should not be block element |

### CODE REVIEW

File: .dumi/theme/common/styles/Markdown.tsx [hunk 1]
```diff
@@ -24,7 +24,6 @@ const GlobalStyle: React.FC = () => {
         }
 
         .markdown img {
-          display: block;
```
Comment (li-jia-nan) [inline .dumi/theme/common/styles/Markdown.tsx:27]: 之前为了消除图片的间隙，加上了 `display: block`，但是对很多地方样式造成了影响，这里回滚一下

**Comment by li-jia-nan** [comment]
@afc163 这回真的没问题了❤️

File: .dumi/theme/common/styles/Markdown.tsx [hunk 2]
```diff
@@ -114,7 +113,8 @@ const GlobalStyle: React.FC = () => {
 
         .markdown ul > li,
         .markdown ol > li {
-          margin-inline-start: ${token.marginLG}px;
+          padding-inline-start: ${token.paddingXXS}px;
+          margin-inline-start: ${token.marginMD}px;
```
Comment (li-jia-nan) [inline .dumi/theme/common/styles/Markdown.tsx:117]: 这里也回滚，跟之前的样式保持一致

**Comment by li-jia-nan** [comment]
@afc163 这回真的没问题了❤️

**Review by afc163 — APPROVED** [review]

**Pull request merged**